MIGRATION_GUIDE.md

```diff
@@ -6,6 +6,11 @@ across different versions.
 
 ## v0.93.0 ➞ v0.94.0
 
+### *(breaking change)* refactored snowflake_schema resource
+
+Changes:
+- renamed `is_managed` to `with_managed_access`
```

Comment:
maybe it would be worth to list added things to the schema (e.g. outputs from queries or OptionalComputed params that ppl can now specify).

Reply:
Done.